frame/collective/src/lib.rs

```diff
@@ -647,15 +668,17 @@ decl_module! {
 				Self::deposit_event(RawEvent::Closed(proposal_hash, yes_votes, no_votes));
 				let (proposal_weight, proposal_count) =
 					Self::do_approve_proposal(seats, voting, proposal_hash, proposal);
-				return Ok(Some(
-					T::WeightInfo::close_approved(len as u32, seats, proposal_count)
-						.saturating_add(proposal_weight)
+				return Ok((
+					Some(T::WeightInfo::close_approved(len as u32, seats, proposal_count)
+					.saturating_add(proposal_weight)),
+					Pays::No
```

Comment:
```suggestion
					Pays::No,
```

Small nit pick detail, we like to put a comma at the end of newline separated lists.

https://users.rust-lang.org/t/trailing-commas/13993

Few more instances of this in other spots in this PR

Reply:
Thanks Shawn, will review again and fix it